src/ansys/units/unit.py

```diff
@@ -657,13 +658,12 @@ def __init__(self, unit1, unit2):
         )
 
 
-class IncorrectTemperatureUnits(ValueError):
-    """Raised when incompatible temperature dimensions are added."""
+class ProhibitedTemperatureOperation(ValueError):
```

Comment:
I think this sould be derived from `TypeError`. This is similar to:
```
>>> "1" / "1"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for /: 'str' and 'str'
```

Reply:
Done. Thank you.

Reply:
Just to confirm, pint does the same as what @mkundu1 is suggesting.

Reply:
@seanpearsonuk Thanks for the confirmation.